src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java

```diff
@@ -32,10 +36,60 @@
     @MockBean
     private OcrDataValidator ocrDataValidator;
 
```

Comment:
Not related with this PR, but we should also verify that the controller returns 200 when request processing results with validation errors (for well-formed requests).

Reply:
Also, not sure if we should return 400 for empty OCR field list - I think this should be left for the validator.

Reply:
I'll raise a new pr for the above

Reply:
checked with @nitinprabhu about the empty OCR field list, it shouldn't be 400 response.